### PR TITLE
add a better doc/description over bundle options in the makefile

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
Describe how to use the target and customize them to build the bundle and its image

**Motivation for the change:**

closes: https://github.com/operator-framework/operator-sdk/issues/4377

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: internal/plugins/manifests/init.go [hunk 1]
```diff
@@ -71,18 +71,32 @@ func initUpdateMakefile(cfg *config.Config, filePath string) error {
 
 // Makefile fragments to add to the base Makefile.
 const (
-	makefileBundleVarFragment = `# Current Operator version
+	makefileBundleVarFragment = `# VERSION defines the project version for the bundle then, update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without change the standard setup, you can:
```
Comment (varshaprasad96) [inline internal/plugins/manifests/init.go:75]:
```suggestion
# To re-generate a bundle for another specific version without changing the standard setup, you can:
```

File: internal/plugins/manifests/init.go [hunk 2]
```diff
@@ -71,18 +71,32 @@ func initUpdateMakefile(cfg *config.Config, filePath string) error {
...
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 VERSION ?= 0.0.1
-# Default bundle image tag
-BUNDLE_IMG ?= controller-bundle:$(VERSION)
-# Options for 'bundle-build'
+
+# CHANNELS defines the bundle channels used in the bundle then, add a new line here if you want to change the it. (E.g CHANNELS = "preview,fast,stable")
+# To re-generate a bundle for others specific channels without change the standard setup, you can:
```
Comment (varshaprasad96) [inline internal/plugins/manifests/init.go:81]:
```suggestion
# CHANNELS define the bundle channels used in the bundle. Add a new line here if you would like to change it. (E.g CHANNELS = "preview,fast,stable")
# To re-generate a bundle for other specific channels without changing the standard setup, you can:
```

File: internal/plugins/manifests/init.go [hunk 3]
```diff
@@ -71,18 +71,32 @@ func initUpdateMakefile(cfg *config.Config, filePath string) error {
...
+# - use the CHANNELS as arg of the bundle target (e.g make bundle CHANNELS=preview,fast,stable)
+# - use environment variables to overwrite this value (e.g export CHANNELS="preview,fast,stable")
 ifneq ($(origin CHANNELS), undefined)
 BUNDLE_CHANNELS := --channels=$(CHANNELS)
 endif
+
+# DEFAULT_CHANNEL defines the default channel used in the bundle then, add a new line here if you want to change it. (E.g DEFAULT_CHANNEL = "stable")
+# To re-generate a bundle for another specific default channel without change the default setup, you can:
```
Comment (varshaprasad96) [inline internal/plugins/manifests/init.go:89]:
```suggestion
# DEFAULT_CHANNEL defines the default channel used in the bundle. Specify the default channel if you like to change the default value (E.g DEFAULT_CHANNEL = "stable")
# To re-generate a bundle for any other default channel without changing the default setup, you can:
```

Just a small doubt - is default channel `stable` or `alpha` if not specified?

Comment (camilamacedo86) [reply internal/plugins/manifests/init.go:89]: Currently yes. But I hope that we can replace that in 2.0.0 for preview. See: https://github.com/operator-framework/operator-sdk/issues/4396

However, the desc says `if you like to change the default value`  so the example cannot be alpha anyway. I added in the description the nomenclature adopted as recommended channels names for OLM.

File: internal/plugins/manifests/init.go [hunk 4]
```diff
@@ -71,18 +71,32 @@ func initUpdateMakefile(cfg *config.Config, filePath string) error {
 
 // Makefile fragments to add to the base Makefile.
 const (
-	makefileBundleVarFragment = `# Current Operator version
+	makefileBundleVarFragment = `# VERSION defines the project version for the bundle. Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without change the standard setup, you can:
```
Comment (jberkhahn) [inline internal/plugins/manifests/init.go:75]: shouldn't this be 'without changing'?

Comment (camilamacedo86) [reply internal/plugins/manifests/init.go:75]: fixed

**Review by varshaprasad96 — APPROVED** [review]

**Review by jmrodri — APPROVED** [review]
We should probably look at adding the `help` target to our generated `Makefile`s in the future. But this is really helpful because I often forget about these.

**Pull request merged**